docs/src/development/local/lando.md

```diff
@@ -4,43 +4,190 @@ sidebarTitle: "Lando"
+Access your app and services by opening the `APP URLS` in your browser.
+
+For more examples, see the [getting started page](https://docs.lando.dev/platformsh/getting-started.html).
+
+## Configuring lando
+
+Lando configuration is located in [the `.lando.yml` file](https://docs.lando.dev/platformsh/config.html).
+
+If you changed your [`.platform.app.yaml`](../../configuration/app/_index.md), [`.platform/routes.yaml`](../../configuration/routes/_index.md) configuration files, run `lando rebuild` for the changes to be taken into account.
+
+### Untrusted SSL certificate
+
+By default, when accessing your local Lando sites through `https` you get an error message in your browser.
+This is expected behavior:
+
+> Lando uses its own Certificate Authority (CA) to sign the certificates for each service and to ensure that these certs are trusted on the internal Lando network.
+However, while Lando will automatically trust this CA internally it's up to you to trust it on your host machine. Doing so alleviates browser warnings regarding the certificates issued.
+
+Find out how to solve it in [Lando's Blog](https://lando.dev/blog/2020/03/20/_5-things-to-do-after-you-install-lando/).
+
+## Common tasks
+
+**TODO: clarify that part + test**
+
+- [Update](https://docs.lando.dev/help/updating.html#_1-do-a-lando-rebuild)
+- [Sync](https://docs.lando.dev/platformsh/sync.html)
+
+More usage examples of the `.lando.yml` file [can be found in GitHub](https://github.com/lando/platformsh/tree/main/examples)
+
+## Troubleshooting
+
+- Make sure that the platform config files (`.platform.app.yaml`, `.platform/routes.yaml`) are present in your local repository.
+- Check that the [services and languages](https://docs.lando.dev/platformsh/config.html#services-yaml) defined in your `.platform.app.yaml` are supported by Lando.
+- Check the [caveats and known issues](https://docs.lando.dev/platformsh/caveats.html).
+- Carefully check the output of the Lando commands you run to spot warnings and errors.
+- Run `lando rebuild`.
+- Restart Lando in debug mode `lando restart -vvv`.
+- Check that you don't face a common issues, as the [dns-rebind](https://docs.lando.dev/help/dns-rebind.html) for example.
+- Check the [Lando documentation](https://docs.lando.dev/help/logs.html#install-logs) for more extensive troubleshooting.
+
+### Access the logs
+
+Run `lando list` to get the list of services.
+Access specific logs with `lando logs -s <service to inspect>`, or access the global logs with `lando logs`.
+
+For more guidance regarding the logs, [check the Lando documentation](https://docs.lando.dev/help/logs.html)
+
+### Something still wrong?
+
+- Get in touch with Lando via [Slack](https://launchpass.com/devwithlando)
+- [Report a bug](https://github.com/lando/platformsh/issues)
```

Comment:
I think all of this could be replaced with this link: https://docs.lando.dev/platformsh/support.html